.github/workflows/check-format.yaml

```diff
@@ -0,0 +1,24 @@
+name: check-format
+on: [pull_request]
+
+jobs:
+  check-format:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 16
+      - name: Install Dependencies
+        run: npm ci
+      - name: Verify format with Prettier
+        uses: EPMatt/reviewdog-action-prettier@v1
+        with:
+          github_token: ${{ secrets.github_token }}
+          reporter: github-pr-review
+          fail_on_error: true
```

Comment:
This step no longer calls our `npm run format:check` script, but it _does_ still use our installed version of prettier. [The significance and importance of this is described by the action's author](https://github.com/EPMatt/reviewdog-action-prettier#why-does-the-action-require-a-packagejson-with-prettier-declared-as-a-dependency).